Unity.gitignore

```diff
@@ -1,9 +1,10 @@
-[Ll]ibrary/
-[Tt]emp/
-[Oo]bj/
-[Bb]uild/
-[Bb]uilds/
-Assets/AssetStoreTools*
+/[Ll]ibrary/
+/[Tt]emp/
+/[Oo]bj/
+/[Bb]uild/
+/[Bb]uilds/
+/Assets/AssetStoreTools*
+/Assets/Temp.meta
```

Comment:
Some people dont even enable .meta in their project, why is this here?

Reply:
now .meta files are on by default, but still don't understand why Temp.meta should exist in this .gitignore